package.json

```diff
@@ -70,21 +70,21 @@
     "es6-promise": "^3.0.2",
     "fast-isnumeric": "^1.1.1",
     "font-atlas-sdf": "^1.3.3",
-    "gl-cone3d": "^1.1.0",
+    "gl-cone3d": "git://github.com/archmoj/gl-cone3d.git#823cabfbb19d8b457730320010cb58df748ff9ee",
```

Comment:
https://github.com/gl-vis/gl-cone3d/compare/master...archmoj:rev-bounds

Reply:
Good question. In the previous version the checks for ranges were commented. When I added the checks similar to the other shaders the cones did not show up. Also for backward compatibility I thought there may be no need for them. But as you mentioned on another comment the are other issues with cones. So I will investigate this.

Reply:
Also added discard logic for cone triangle-fragment.glsl as well as streamtubes.